.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  // Verwendet IntelliSense zum Ermitteln möglicher Attribute.
+  // Zeigen Sie auf vorhandene Attribute, um die zugehörigen Beschreibungen anzuzeigen.
+  // Weitere Informationen finden Sie unter https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Rails server",
+      "type": "Ruby",
+      "request": "launch",
+      "program": "${workspaceRoot}/bin/rails",
+      "args": [
+        "server"
+      ],
+      "useBundler": true
+    }
+  ]
+}
```

Comment:
die datei sollte nicht hier commited werden. am besten in globale gitignore aufnehmen.

Reply:
erledigt

Reply:
hast du jetzt aber ins projekt gepackt und nicht in deine globale gitignore. so kann es sein, dass es bei einem nächsten projekt wieder auftreten wird.